_posts/2021-11-18-keras-updates/keras-updates.Rmd

```diff
@@ -0,0 +1,159 @@
+---
+title: "Keras for R is back!"
+description: > 
+  For a while, it may have seemed that Keras for R were in some undecidable state, like Schrödinger's cat before inspection. It is high time to  correct that impression. Keras for R is back, with two recent releases adding powerful capabilities that considerably lighten previously tedious tasks. 
```

Comment:
```suggestion
  For a while, it may have seemed that Keras for R was in some undecidable state, like Schrödinger's cat before inspection. It is high time to  correct that impression. Keras for R is back, with two recent releases adding powerful capabilities that considerably lighten previously tedious tasks. 
```

Reply:
I actually had a whole discussion about this yesterday. I didn't notice anything wrong with this when I first read it. But then when another person did say something, I began to doubt my native-english-speaker abilities and wonder "why didn't I catch that?". Especially since the "was" version still didn't quite look right to my eyes. After some googling I think I convinced myself that "were" is more appropriate, since it refers to a imagined or doubtful thing: https://www.grammarly.com/blog/was-vs-were/

Reply:
Yeah, this one's all sprachgefühl. I'm pretty sure that I just steer around describing my imaginings in French—the verb conjugation is too much for me! 😝

Reply:
so enlighten me please - why "was"? I was master of subjonctif in school but my French doesn't help me here  ;-)))

(my choosing _were_ was just due to the fact that i only learned this few years ago, and i love it ;-)))

Reply:
It just sounds off. Like, I get that it’s a bit of a grammatical niche/wiggle spot, but it makes it sound like you should be looking for two things (or plural) instead of “R for Keras.” Subjunctive isn’t used a lot in English. Your call. It’s a complex verb phrase you’ve constructed there “seemed as though it…” Given it’s the first line of the description/summary blurb that will show up with the post (both in the blog listings, and in the metadata/social sharing cards), you don’t want to bury the meat of what the post is about.

Reply:
I already committed your suggestion ... :-)  In any case, I get that it's harder to parse, so not great for a summary